### PR TITLE
[LG-5313] Redirect uri wildcard validations

### PR DESCRIPTION
**Pull request by TiannaW24**
### Work Ticket
https://cm-jira.usa.gov/browse/LG-5313

### Changes

- Add validations and error message when SPs try to use redirect uris with wildcards
- Update unit tests

### PR Checklist:

1. Have you tagged the appropriate dev(s) for review?
2. Have you moved any Jira/Smartsheet tickets to Code Review?

### CODE REVIEW

File: app/models/service_provider.rb [hunk 1]
```diff
@@ -180,4 +181,16 @@ def contains_invalid_attribute?
     possible_attributes = ALLOWED_IAL1_ATTRIBUTES + ALLOWED_IAL2_ATTRIBUTES
     attribute_bundle.any? { |att| !possible_attributes.include?(att) }
   end
+
+  def redirect_uris_contains_no_wildcards
```
Comment (TiannaW24) [inline app/models/service_provider.rb:185]: I dont like repeating this code just bc the validator gem is generic. I'll be going back and adding specificity to the errors returned from the validator. Hold off on review.

Comment (TiannaW24) [reply app/models/service_provider.rb:185]: Refactor incoming: https://github.com/18F/identity-validations/pull/19

**Comment by TiannaW24** [comment]
Screen Shot 2022-01-28 at 12 33 18 PM

**Review by jauschalley — APPROVED** [review]
This is so beautiful 😢

**Review by russbeye — APPROVED** [review]
Great work, and a much better experience for our partners! 👏

**Review by orenyk — COMMENTED** [review]
Great work!

**Pull request merged**